lib/puma/control_cli.rb

```diff
@@ -141,6 +141,12 @@ def send_request
 
       # create server object by scheme
       server = case uri.scheme
+                when "ssl"
+                  require 'openssl'
```

Comment:
I'm not totally sure if this was the right thing to have done here. I started out with:

```sh
require 'puma/puma_http11'
require 'puma/minissl'
MiniSSL.check
MiniSSL::Socket.new(
  TCPSocket.new(uri.host, uri.port),
  MiniSSL::Engine.client
)
```

but that was raising an "Unknown OpenSSL error: 2" at https://github.com/puma/puma/blob/3294c82037f60da8c5dc810a7ff3a61ed11ee707/ext/puma_http11/mini_ssl.c#L324 and both my C and SSL knowledge are probably a bit too limited to know what to do with that without some help.